languages/fsrs4tw.multids

```diff
@@ -1,57 +1,57 @@
 title: $:/language/tidme/
 
-tidme: 墨屉
```

Comment:
Same with this: https://github.com/oflg/Tidme-manual-zh-Hans/pull/1/commits/6985a0138be32881079cb0c5fc0208fcb48b277d#r1388914167